src/Expenses/expensesControllers.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const expensesServices = require('./expensesServices');
+
+const getAllExpenses = (req, res) => {
+  res.send(expensesServices.getAll());
+};
+
+const getOneExpense = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (!expenseId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundExpense = expensesServices.getById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const addExpense = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const hasBody = userId && spentAt && title && amount && category && note;
```

Comment:
According to documentation `note` is not obligatory field)

Reply:
Yeah, thnx